tests/config/dynamic.tests.yaml

```diff
@@ -29,6 +29,7 @@ a:
         pool: ams
       - geos:
           - NA-US-CA
+          - NA-US-NC
```

Comment:
Any specific reason for the NC addition?

Reply:
Pure romanticism, since that's where I live. I'll happily yank it back out if you'd prefer.